pkg/vetter/conflictingvirtualservicehost/vet.go

```diff
@@ -91,6 +104,14 @@ func createVirtualServiceNotes(virtualServices []*v1alpha3.VirtualService) ([]*a
 	return notes, nil
 }
 
+func appendNotes(hg hostAndGateway, vs *v1alpha3.VirtualService, vsByHostAndGateway VirtualSvcByHostAndGateway) {
```

Comment:
`appendNotes` is a bit misleading, you're just populating your map here. I would suggest calling this function `populateVirtualServiceMap`.

Reply:
True! thanks!